doc-content/enterprise-only/openshift/hacep-server-proc.adoc

```diff
@@ -32,14 +41,27 @@ oc create -f kubernetes/role-binding.yaml
 +
 For more information about configuring role-based access control in {OPENSHIFT}, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html/authentication/using-rbac[Using RBAC to define and apply permissions] in the {OPENSHIFT} product documentation.
 +
-. In the `springboot` directory, enter the following commands to build the Docker image and push it into the Docker registry that is configured on your system. (Consider configuring a private registry before running these commands). This build imports the built `sample-hacep-project-kjar` code as a Maven dependency and includes it in the `BOOT-INF/lib` directory of the `openshift-kie-springboot.jar` file. The Docker build then uses the JAR file to create the image.
+. In the `springboot` directory, enter the following commands to create the image for deployment and push it into the repository configured for your OpenShift environment:
++
+----
+oc new-build --binary --strategy=docker --name openshift-kie-springboot
+oc start-build openshift-kie-springboot --from-dir=. --follow
+----
++
+. Enter the following command to detect the name of the image that was built:
 +
 ----
-docker login --username=<user username>
-docker build -t <user_username>/openshift-kie-springboot:<tag> .  
-docker push <user_username>/openshift-kie-springboot:<tag>
+oc get is/openshift-kie-springboot -o template --template='{{range .status.tags}}{{range .items}}{{.dockerImageReference}}{{end}}{{end}}'
 ----
 +
-Alternatively, to use Podman for the build process, replace `docker` with `podman` in the commands. 
+. Edit the `kubernetes/deployment.yaml` file. Replace the existing image URL with the result of the previous command, removing all characters at the end starting with the `@` symbol, then add `:latest` to the line. For example: 
```

Comment:
Similar to before. Try to make one sentence or make two steps. In this case, consider:
Open the `kubernetes/deployment.yaml` file in a text editor.
Replace...

Reply:
Well, you know my opinion on For example: Should be a sentence.